### PR TITLE
added call back for getting vendor/product id

### PR DESCRIPTION
**Pull request by frankmorgner**
implementation taken from
https://github.com/jasp00/OpenSC/commit/83142d4caeae5958850795757c7d6c5cf072a15c

### CODE REVIEW

**Comment by frankmorgner** [comment]
Maybe you're right...

**Comment by frankmorgner** [comment]
looking through use cases of vid/pid I stumbled across https://www.opensc-project.org/opensc/ticket/340. Since the PIN recognition of this reader has been fixed in @LudovicRousseau's CCID driver and the MS CCID driver does not support PIN entry, I think the OpenSC code for handling this reader can be removed, right?

**Comment by frankmorgner** [comment]
I updated the commit for internal usage only. currently the vendor/prduct id is only debugged. in the future it could be used for implementing workarounds of reader bugs

File: src/libopensc/reader-ctapi.c [hunk 1]
```diff
@@ -561,7 +561,7 @@ struct sc_reader_driver * sc_get_ctapi_driver(void)
 	ctapi_ops.perform_verify = ctbcs_pin_cmd;
 	ctapi_ops.perform_pace = NULL;
 	ctapi_ops.use_reader = NULL;
-
+
```
Comment (martinpaljak) [inline src/libopensc/reader-ctapi.c]: Aha!

**Comment by frankmorgner** [comment]
I rebased the commits onto the current master.

I also removed handling the HP reader as it is solved already by libccid.

**Pull request merged**